packages/cashier/src/Components/Error/cashier-locked.jsx

```diff
@@ -61,120 +62,93 @@ const CashierLocked = ({
-                ]}
-            />
-        );
-    } else if (ask_tin_information) {
-        message = (
-            <Localize
-                i18n_default_text='You have not provided your tax identification number. This information is necessary for legal and regulatory requirements. Please go to <0>Personal details</0> in your account settings, and fill in your latest tax identification number.'
-                components={[
-                    <a key={0} className='link' rel='noopener noreferrer' href={'/account/personal-details'} />,
-                ]}
-            />
-        );
-    } else if (ask_uk_funds_protection) {
-        message = (
-            <Localize
-                i18n_default_text='Your cashier is locked. See <0>how we protect your funds</0> before you proceed.'
-                components={[<a key={0} className='link' rel='noopener noreferrer' href={'/cashier/deposit'} />]}
-            />
-        );
-    } else if (ask_self_exclusion_max_turnover_set) {
-        message = (
-            <Localize
-                i18n_default_text='Your access to Cashier has been temporarily disabled as you have not set your 30-day turnover limit. Please go to <0>Self-exclusion</0> and set your 30-day turnover limit.'
-                components={[<a key={0} className='link' rel='noopener noreferrer' href={'/account/self-exclusion'} />]}
-            />
-        );
-    } else if (ask_fix_details) {
-        if (is_deposit_lock) {
-            icon = 'IcCashierDepositLock';
-            title = localize('Deposits are locked');
+        } else if (ask_uk_funds_protection) {
             message = (
                 <Localize
-                    i18n_default_text='Your <0>personal details</0> are incomplete. Please go to your account settings and complete your personal details to enable deposits.'
-                    components={[
-                        <a key={0} className='link' rel='noopener noreferrer' href={'/account/personal-details'} />,
-                    ]}
+                    i18n_default_text='Your cashier is locked. See <0>how we protect your funds</0> before you proceed.'
+                    components={[<a key={0} className='link' rel='noopener noreferrer' href={'/cashier/deposit'} />]}
                 />
             );
-        } else if (is_withdrawal_lock) {
-            icon = 'IcCashierWithdrawalLock';
-            title = localize('Withdrawals are locked');
+        } else if (ask_self_exclusion_max_turnover_set) {
             message = (
                 <Localize
-                    i18n_default_text='Your <0>personal details</0> are incomplete. Please go to your account settings and complete your personal details to enable withdrawals.'
+                    i18n_default_text='Your access to Cashier has been temporarily disabled as you have not set your 30-day turnover limit. Please go to <0>Self-exclusion</0> and set your 30-day turnover limit.'
```

Comment:
Looks like we're using the word `cashier` with small `c` everywhere else. Is this intentional?

Reply:
I believe it's expected. The messages were provided by content team and I haven't done any change to it on this PR.